app/Http/Requests/ProgramRequest.php

```diff
@@ -21,10 +23,40 @@ public function authorize()
      *
      * @return array
      */
-    public function rules()
+    public static function rulesStore($request)
     {
-        return [
-            'name' => 'required|unique:programs,name',
+        $messages = [
+            'name.required' => __('Program') . __('Required'),
+            'name.string' => __('Program') . __('String'),
+            'name.unique' => __('Program') . __('Unique'),
         ];
+        
+        return  Validator::make($request->all(), [
+                'name' => 'required|string|unique:programs',
+            ], $messages
+        );
+    }
+
+    public static function rulesUpdate($request, $id)
+    {
+        $messages = [
+            'name.required' => __('Program') . __('Required'),
+            'name.string' => __('Program') . __('String'),
+            'name.unique' => __('Program') . __('Unique'),
+        ];
+
+        $program = Program::findOrFail($id);
+        
+        if($program->name == $request->name){
+            return  Validator::make($request->all(), [
+                    'name' => 'required|string',
+                ], $messages
+            );
+        } else if ($program->name != $request->name) {
+            return  Validator::make($request->all(), [
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.LanguageConstructSpacing.IncorrectSingle</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 56</br>Language constructs must be followed by a single space; expected 1 space but found "··"